Readme.md

```diff
@@ -75,6 +76,17 @@ var HashMap = require('hashmap');
 map.set("some_key", "some value");
 map.get("some_key"); // --> "some value"
 ```
+
+### Map size / number of elements
+
+```js
+var map = new HashMap();
+map.set("key1", "val1");
+map.set("key2", "val2");
+map.size; // -> 2
+map.count(); // Alias -> 2

```

Comment:
If we do deprecate `count()`, better to stop mentioning it asap. I say just delete this line